script.js

```diff
@@ -58,15 +69,51 @@ class Bookshelf {
     if (title.value !== '' && author.value !== '') {
       const book = new Book(title.value, author.value);
       this.books.push(book);
-      this.savedata();
+      this.#savedata();
       bookShelf.innerHTML = '';
       this.retrievedata();
     }
   }
 }
 
+function hidepages(event) {
+  switch (event.target.id) {
+    case 'list':
+      if (!addNewDisplay.classList.contains('hidden')) addNewDisplay.classList.add('hidden');
+      if (!contactDisplay.classList.contains('hidden')) contactDisplay.classList.add('hidden');
+      if (listDisplay.classList.contains('hidden')) listDisplay.classList.remove('hidden');
+      if (!listBtn.classList.contains('rightmenuSelected')) listBtn.classList.add('rightmenuSelected');
+      if (addNewBtn.classList.contains('rightmenuSelected')) addNewBtn.classList.remove('rightmenuSelected');
+      if (contactBtn.classList.contains('rightmenuSelected')) contactBtn.classList.remove('rightmenuSelected');
+      break;
+
+    case 'addNew':
+      if (!listDisplay.classList.contains('hidden')) listDisplay.classList.add('hidden');
+      if (!contactDisplay.classList.contains('hidden')) contactDisplay.classList.add('hidden');
+      if (addNewDisplay.classList.contains('hidden')) addNewDisplay.classList.remove('hidden');
+      if (!addNewBtn.classList.contains('rightmenuSelected')) addNewBtn.classList.add('rightmenuSelected');
+      if (listBtn.classList.contains('rightmenuSelected')) listBtn.classList.remove('rightmenuSelected');
+      if (contactBtn.classList.contains('rightmenuSelected')) contactBtn.classList.remove('rightmenuSelected');
```

Comment:
- [ ] (Optional) When you finish adding a book, you can display a message to say the book has been added and then tell the user to click the button that navigates to the list of books. Another approach is to change the page automatically to the list of books page after a book has been added. 👍